spec/models/spree/product_spec.rb

```diff
@@ -714,4 +714,45 @@ module Spree
       end
     end
   end
+
+  describe "product import" do
+    describe "finding the most recent import date of the variants" do
+      let!(:product) { create(:product) }
+
+      let(:reference_time) { Time.zone.now.beginning_of_day }
+
+      before do
+        product.reload
```

Comment:
why a reload if we just create the product? Do we really need it?

Reply:
Yes, this was to uncache the initially empty `product.variants` which was getting cached somewhere during setup of the product. :slightly_smiling_face: I prefer this over reloading just `product.variants`, which could leave you with outdated fields, e.g. (hypothetical) `order.total` after adding to `order.line_items`.

Reply:
@sauloperez Can I merge this now? :heavy_check_mark: :thinking: